mne/utils/progressbar.py

```diff
@@ -37,17 +37,23 @@ class ProgressBar(object):
         a sane value based on the current terminal width.
     max_value : int | None
         The max value. If None, the length of ``iterable`` will be used.
+    which_tqdm : str
```

Comment:
```suggestion
    which_tqdm : str | None
```
By local convention `None` is for us a wrapper to "use the config value"